Classes/Composer/InstallerScripts.php

```diff
@@ -71,8 +72,10 @@ public static function postUpdateAndInstall(Event $event)
             echo '> Version:      ' . $version . $versionMessage . "\n";
             echo '> Platform:     ' . $platform . "\n";
             echo '> Architecture: ' . $architecture . "\n";
+
+            $httpClient = new Client();
             $downloadLink = sprintf('https://github.com/Flowpack/prunner/releases/download/v%s/prunner_%s_%s_%s.tar.gz', $version, $version, $platform, $architecture);
-            $downloadedFileContents = file_get_contents($downloadLink);
+            $downloadedFileContents = $httpClient->get($downloadLink)->getBody()->getContents();
```

Comment:
We could optimize this if we directly open a stream to the target file and call Guzzle via `$client->request('GET', $downloadLink, ['save_to' => $stream]);`

Reply:
now using the `sink` property to download directly to file